dashboard/app/assets/stylesheets/application.scss

```diff
@@ -115,6 +115,11 @@ th {
   padding: 5px;
 }
 
+td {
```

Comment:
Is this still needed? I'm always a bit nervous about changing `application.scss`